src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/events/EventListener.kt

```diff
@@ -13,15 +16,17 @@ import java.io.IOException
 import java.util.function.Consumer
 
 @Service
-@ConditionalOnProperty(name = ["sqs.provider"])
 class EventListener(
   private val nomisService: NomisService,
   private val pollPrisonersService: PollPrisonersService,
   private val objectMapper: ObjectMapper
 ) {
-  @JmsListener(destination = "\${sqs.events.queue.name}")
-  fun eventListener(requestJson: String) {
-    val event = getOffenderEvent(requestJson)
+  @SqsListener("events", factory = "hmppsQueueContainerFactoryProxy")
+  @Throws(JsonProcessingException::class)
+  fun onOffenderEvent(message: String, attributes: QueueAttributes) {
+
+    val event = getOffenderEvent(message)
+
```

Comment:
ktlint does not like the old code that has an if without an else, so i have added blank elses for now